backend/src/revisions/revisions.service.spec.ts

```diff
@@ -141,26 +142,57 @@ describe('RevisionsService', () => {
     });
 
     it('purges the revision history', async () => {
-      const revision1 = Mock.of<Revision>({ id: 1 });
-      const revision2 = Mock.of<Revision>({ id: 2 });
-      const revision3 = Mock.of<Revision>({ id: 3 });
+      const revision1 = Mock.of<Revision>({
+        id: 1,
+        note: Promise.resolve(note),
+      });
+      const revision2 = Mock.of<Revision>({
+        id: 2,
+        note: Promise.resolve(note),
+      });
+      const revision3 = Mock.of<Revision>({
+        id: 3,
+        note: Promise.resolve(note),
+        content: '---\ntitle: new title\ndescription: new description\ntags: [ "tag1" ]\n---\nnew content\n',
+      });
       revisions = [revision1, revision2, revision3];
       note.revisions = Promise.resolve(revisions);
 
       jest.spyOn(revisionRepo, 'find').mockResolvedValueOnce(revisions);
       jest.spyOn(service, 'getLatestRevision').mockResolvedValueOnce(revision3);
 
+      // expected to have only the latest revision
+      revision3.patch = createPatch(
+        note.publicId,
+        '',
+        revision3.content,
+      )
+      jest.spyOn(revisionRepo, 'save').mockResolvedValue(revision3)
+
       // expected to return all the purged revisions
       expect(await service.purgeRevisions(note)).toStrictEqual([
         revision1,
         revision2,
       ]);
 
-      // expected to have only the latest revision
       expect(revisions).toStrictEqual([revision3]);
+      expect(revision3.patch).toMatchInlineSnapshot(`
+        "Index: test-note
+        ===================================================================
+        --- test-note
+        +++ test-note
+        @@ -0,0 +1,6 @@
+        +---
+        +title: new title
+        +description: new description
+        +tags: [ "tag1" ]
+        +---
+        +new content
+        "
+      `);
```

Comment:
Maybe we could also just use a "normal" snapshot instead of an embedded one. Seems easier in terms of maintenance to me (updating via `jest -u`) and is more aligned with the rest of the tests.

Reply:
Thank you for your review!!
I had changed snapshot type from inline to file like frontend.